### ISSUE TITLE
Boxart & Gameart

### ISSUE THREAD

**Issue by wiz122** [MrZammler/iGame#137]
Would it be possible to add an option to have two windows, one for boxart and the other for game screens in a future release?

Vampire or fast accelerator users should see no slowdown or lag whilst loading two pictures together but maybe there could be an enable/disable option in the menu too so people using a basic/slower system won't see a performance hit.

**Comment by walkero-gr** [comment]
Thank you @wiz122 for this proposal. Do you recommend this to be a separated window or included in the main window?
For sure this should have a way to be disabled so that users with not so powerful machines will not have to wait for a long time to show the game on selection.
Do you see this as necessary when the user selects a game or as a window that opens and shows extra info for the game, when the user presses an info button?

**Comment by MrV2K** [comment]
I think that the main problem on an Amiga is screen space. On AGA/ECS at a max of 640x512, you'd have to lose the genres list to fit the two windows. You could get past this by switching out the genres list to a clickable dropdown list (if MUI supports that) It may be better to have them as a tab or a menu option so you can switch between them. The other problem is the naming. You'd need to add an extension to the igame.iff files so it could differentiate between image types. TinyLauncher gets over the problem by using the "SCR0", "SCR1" extensions. The other option would be to store the images local to IGame in their own folder based on type.

This is my IGame inspired WHD launcher for Windows as an example.

![UltraMiggy](https://user-images.githubusercontent.com/71010565/110480617-983a8880-80de-11eb-9ba8-8d8b79b7289b.png)

**Comment by wiz122** [comment]
Yes, something similar to that picture is what I would like to see but perhaps Cover could be displayed in the top window and the Screenshot underneath. Currently iGame looks for igame.iff but could each window look specifically for a certain .iff? for example the Cover window look for cover.iff and Screenshot window look for screenshot.iff to differentiate between them?

Perhaps even both windows could be locked to the same dimensions to avoid loosing the Genres list underneath? Personally though I never use the Genres list (or have known how to use it) so wouldn't mind if did become a menu or tab option.

**Comment by walkero-gr** [comment]
Thanks guys for the info. What I try to understand is the purpose. The game screenshot is necessary to understand a game, and that's why iGame author @MrZammler added it. The cover image is more like a nice to have info about the game, like so many others, like the Publisher, the Year etc.. So, would make sense to have the cover opening in a new window when the user clicks a button to see more info about the game?

@MrV2K The UltraMiggy looks awesome. Where can I find more info? Are you aware of the https://blitterstudio.com/amigula/ ?

**Comment by MrV2K** [comment]
> Thanks guys for the info. What I try to understand is the purpose. The game screenshot is necessary to understand a game, and that's why iGame author @MrZammler added it. The cover image is more like a nice to have info about the game, like so many others, like the Publisher, the Year etc.. So, would make sense to have the cover opening in a new window when the user clicks a button to see more info about the game?
> 
> @MrV2K The UltraMiggy looks awesome. Where can I find more info? Are you aware of the https://blitterstudio.com/amigula/ ?

@walkero-gr It's a program I wrote for myself. It can load Amiga created IGame game lists, convert the names, add more detailed genres and save them to new 100% IGame compatible game lists (no more Alt Alt Alt). It uses an own made JSON database of all the whdload games to do this and a custom made genres file. It can also download WHDLoad games from FTP and update it's amiga HDF file with the new/updated games. I also use it to make the IFF covers and screen shots I've uploaded on EAB. The long term goal it is to have a one stop Amiga frontend. At the moment I'm working on adding a CD32 launcher for it.

**Comment by DisasterIncarnate** [comment]
no need for extra screenspace surely, cant the screetshot area just be used like a slideshow if any more pictures are present?

If a central database/text file was made for the full amiga gamelist, you could include links to say Lemon Amiga, which already has a load of screenshots as well as docs/manuals for the games which could be utilised, dont need to make more of something when theres a ready made source out there already

**Comment by IvanEBC** [comment]
But most games don’t work if the Amiga is net connected… so it’s counter productive.

Ivan Wheelwright
***@***.***

**Comment by DisasterIncarnate** [comment]
hmm yeah suppose, i keep thinking of people using them in modern days where a lot do pimp out their setups with extra stuff like net access and proper or pistorm like upgrades.

**Comment by IvanEBC** [comment]
> hmm yeah suppose, i keep thinking of people using them in modern days where a lot do pimp out their setups with extra stuff like net access and proper or pistorm like upgrades.

I found out very recently, the way around this is built into S: WHDload-Startup and Cleanup where you can turn off the net and have it restart again when exitting... among other handy things.
